Doc/whatsnew/3.15.rst

```diff
@@ -75,7 +75,30 @@ New features
 Other language changes
 ======================
 
-
+* Python now uses UTF-8_ as the default encoding, independent of the system's
```

Comment:
You might mention the UTF-8 Mode earlier since it has other side effects documented in the UTF-8 Mode section, such as changing sys.stdout error handler and ignoring the locale encoding.

Reply:
"default encoding" is UTF-8 since Python 3.0. (e.g. `sys.getdefaultencoding()`)